README.md

```diff
@@ -94,6 +94,12 @@ It's built on top of **RNE & Open Collective**. If you have any feedback or
 would like to know more details about this kindly contact
 **[Monte Thakkar](https://github.com/Monte9)**.
 
+## React Native Web support
+
+As a cross platform UI Toolkit, you can now use RNE on the web & share your codebase between your React Native + React web apps. RNE components are rendered perfectly on browser. You can achieve this to target iOS, Android and Web by collaborating RNE and [React Native for Web](https://github.com/necolas/react-native-web).
+
+Full documentation RNE x RNW, click [here](https://github.com/haruelrovix/react-native-elements/blob/feature/RNExRNW/docs/rnw_support.md).
```

Comment:
this path needs to be updated to link to `docs/rnw_support.md`

Reply:
@Monte9 updated via [c2d06e2](
https://github.com/react-native-training/react-native-elements/pull/1664/commits/c2d06e25075c74106baf208f0c2fcb7d50bca14b)